templates/taxi/car_list.html

```diff
@@ -1,16 +1,32 @@
-{% extends "base.html" %}
+{% extends 'base.html' %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="float: right;" href="{% url 'taxi:car-create' %}">+</a></h1>
```

Comment:
Ensure that the URL pattern `taxi:car-create` is correctly defined in your `urls.py` file, as it is used here to link to the car creation page.